lib/init.js

```diff
@@ -8,7 +8,9 @@ function init (hoodie) {
   account.on('signout', function () {
     // TODO: prevent data loss on sign out:
     //       https://github.com/hoodiehq/hoodie-client/issues/22
-    store.clear()
+    store.reset({
+      name: hoodie.account.id
+    })
```

Comment:
Thanks for the clarification of `hoodie.account.id` makes things clearer now!

I'll update the patch with your feedback.